packages/design-system/src/contexts/popup/context.ts

```diff
@@ -19,4 +19,9 @@
  */
 import { createContext } from '@googleforcreators/react';
 
-export default createContext({});
+export interface PopupContext {
+  topOffset?: number;
+  leftOffset?: number;
```

Comment:
I made these non-optional in my PR to make it easier to consume, but probably doesn't mean a lot at this stage.